spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+    Test.setup_models
+    Rails.application.load_seed
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, js: true) do
+    DatabaseCleaner.strategy = :truncation
```

Comment:
Yes it does mean capybara is always used. 

But there are specs that use capybara where js is not true.  

Honestly, I copied some of the setup from another Rails project you might be familiar with: https://github.com/18F/e-manifest/blob/master/spec/support/database_cleaner.rb

Seems to work there to have different strategies for js vs non-js so I think this should be fine.